sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -757,7 +1189,7 @@ where
 ///
 /// A collection of positions/offsets and length to omit from hashing.
 fn get_object_locations_from_stream<T>(
-    otf_io: &OtfIO,
+    sfnt_io: &SfntIO,
```

Comment:
This function's comments are missing a `sfnt_io` argument comment

Reply:
I tried but it made wigglies. We swarm?

Reply:
Not sure why a comment would make squigglies (I get a "misspelled word" squiggly when I try, but not code squiggly)?

Reply:
anyway the old code didn't have it AND I don't really understand what that thing is - a module handle?

Reply:
maybe just `sfnt_io - SFNT wrapper allowing retrieval/manulation of C2PA data` ?

Reply:
Adding rustdoc stuff to both modules.